### PR TITLE
feat(graph) Add some graphQL fields for UI purpose

### PR DESCRIPTION
**Pull request by ansmonjol**
## Roadmap Task

👉  https://github.com/getlago/lago/issues/{{NUMBER}}

## Context

This PR is part of the credit notes work and will be released on the feature branch

## Description

This PR adds some graphQL fields needed for UI porpose

### CODE REVIEW

File: app/graphql/types/fees/object.rb [hunk 1]
```diff
@@ -20,6 +20,12 @@ class Object < Types::BaseObject
 
       field :creditable_amount_cents, Integer, null: false
 
+      field :amount_currency, Types::CurrencyEnum, null: false
+
+      def amount_currency
+        object.currency
```
Comment (vincent-pochet) [inline app/graphql/types/fees/object.rb:26]: Actually, this fields is already defined in `/app/graphql/types/invoices/invoice_item.rb` which is implemented by this object

File: app/graphql/types/credit_notes/object.rb [hunk 2]
```diff
@@ -37,13 +37,24 @@ class Object < Types::BaseObject
 
       field :file_url, String, null: true
 
+      field :customer, Types::Customers::Object, null: false
       field :invoice, Types::Invoices::Object
       field :items, [Types::CreditNoteItems::Object], null: false
 
+      field :is_credited, Boolean, null: false
+      field :is_refunded, Boolean, null: false
```
Comment (vincent-pochet) [inline app/graphql/types/credit_notes/object.rb:45]: Do you have a need for these fields?

The methods they rely on are defined like this:

```ruby
  def credited?
    credit_amount_cents.positive?
  end

  def refunded?
    refund_amount_cents.positive?
  end
```

and both `refund_amount_cents` and `credit_amount_cents` are already exposed

Comment (ansmonjol) [reply app/graphql/types/credit_notes/object.rb]: I could remove them indeed ! Done

File: app/graphql/types/fees/object.rb (hunk 1 above)
Comment (ansmonjol) [reply app/graphql/types/fees/object.rb:26]: indeed, removed

**Review by vincent-pochet — APPROVED** [review]

**Pull request merged**